recipes/libgeoip/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "GeoIP" %}
+{% set version = "1.6.11" %}
+{% set sha256 = "8859cb7c9cb63e77f4aedb40a4622024359b956b251aba46b255acbe190c34e0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/maxmind/geoip-api-c/archive/v{{ version }}.tar.gz
```

Comment:
Use the official tarball from github releases, https://github.com/maxmind/geoip-api-c/releases/download/v1.6.11/GeoIP-1.6.11.tar.gz